esmvaltool/config-references.yml

```diff
@@ -377,6 +385,15 @@ authors:
   vanulft_bert:
     name: van Ulft, Bert
     institute: KNMI, Netherlands
+    email: bert.van.ulft 'at' knmi.nl
+  verhoeven_stefan:
```

Comment:
Could you also remove the (now) duplicate and wrong entry for Stefan on line 398?